portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that gets data from a csv and parses it to display in a chart.*/
+@WebServlet("/chart-data")
+public class ChartServlet extends HttpServlet {
+
+  // Ordered map that stores search trends for the word 'meme' keyed by year.
```

Comment:
Sorry one last nitpick, I didn't realize it was keyed by year _and_ month, make sure to clarify that!

Reply:
Don't worry, done!